src/d/actor/d_a_hmlif.cpp

```diff
@@ -5,9 +5,45 @@
 
 #include "d/dolzel_rel.h" // IWYU pragma: keep
 #include "d/actor/d_a_hmlif.h"
+#include "d/res/res_hmlif.h"
+#include "d/res/res_hyuf1.h"
+#include "d/res/res_hyuf2.h"
 #include "d/d_procname.h"
 #include "d/d_priority.h"
-#include "d/d_cc_d.h"
+#include "d/d_path.h"
+#include "d/d_com_inf_game.h"
+#include "f_op/f_op_actor_mng.h"
+
+char* daHmlif_c::m_arcname[] = {"Hmlif", "Hyuf1", "Hyuf2"};
+const f32 daHmlif_c::m_speed[] = {
+    1.6666666f,
+    3.3333333f,
+    5.0f,
+    6.6666665f,
+    8.333333f,
+    10.0f,
+    11.666667f,
+    13.333333f,
+    15.0f,
+    16.666666f,
+    18.333334f,
+    20.0f,
+    21.666666f,
+    23.333334f,
+    25.0f,
+    26.666666f,
+};
+const s16 daHmlif_c::m_bmdidx[] = {HMLIF_BDL_HMLIF, HYUF1_BDL_HYUF1, HYUF2_BDL_HYUF2};
+const s16 daHmlif_c::m_dzbidx[] = {HMLIF_DZB_HMLIF, HYUF1_DZB_HYUF1, HYUF2_DZB_HYUF2};
+const s16 daHmlif_c::m_btpidx[] = {-1, HYUF1_BTP_HYUF1, HYUF2_BTP_HYUF2};
+const s16 daHmlif_c::m_brkidx[] = {HMLIF_BRK_HMLIF, HYUF1_BRK_HYUF1, -1};
+
+const f32 daHmlif_c::m_cull_box[][6] = {
+    {-200.0f, -100.0f, -200.0f, 200.0f, 50.0f, 200.0f},
+    {-200.0f, -100.0f, -200.0f, 200.0f, 450.0f, 200.0f},
+    {-200.0f, -100.0f, -200.0f, 200.0f, 450.0f, 200.0f},
+};
```

Comment:
Does this still match when changed to an array of fopAc_cullSizeBox? Like in f_op_actor_mng.cpp